doc/release-notes.md

```diff
@@ -102,6 +102,22 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
+- The `getpeerinfo` RPC no longer returns the `inbound` or `addnode` fields by
```

Comment:
I think it's premature to be deprecating these fields. It would be preferable to let the new code mature for a release or so and to implement how it might be presented in the GUI, before considering removing access to these via the RPC.

Reply:
Agree that we don't need to be overly aggressive with deprecation. If deprecation/removal is too controversial for now, it can be split up into another follow-up pull for later. I think adding the field should be uncontroversial and conceptually easier to get in.

Reply:
I don't think there is a reason to remove "inbound", ever. It's very useful to filter RPC output by on its own.

Reply:
sounds good! I removed the "deprecate inbound" commit. I currently have left in the "deprecate getpeerinfo addnode" commit, but can also remove that one if reviewers would like. I do find the current way of calling it `addnode` (as is in master) misleading because it also returns `-connect` peers. But if having this information separately is desirable, I'm happy to remove that commit as well. I just don't want the default to be the RPC bloating over time with redundant information.

Reply:
> makes two of the existing fields (inbound & addnode booleans) redundant

> sounds good! I removed the "deprecate inbound" commit. I currently have left in the "deprecate getpeerinfo addnode" commit, but can also remove that one if reviewers would like. I do find the current way of calling it `addnode` (as is in master) misleading because it also returns `-connect` peers. But if having this information separately is desirable, I'm happy to remove that commit as well.

If `addnode` has different behavior, which users may depend on, is it redundant?

Reply:
it's not different behavior. its the same. 

`getpeerinfo` field `addnode` returns a bool that indicates if the connection was manually added, either via `addnode` RPC or `-connect` command line argument. 

with this PR, `getpeerinfo` field `connection_type` will return `manual` for these connections.

I'm saying that #2 is more clear and accurate. I think your confusion around possible difference of behavior supports my point that it is unclear :)

Reply:
Ok, I checked the code and indeed, (a) I misread you and (b) getpeerinfo `addnode` and `conn_type manual` are the same.